src/storage/AddVerticesProcessor.cpp

```diff
@@ -22,30 +22,133 @@ void AddVerticesProcessor::process(const cpp2::AddVerticesRequest& req) {
     version = folly::Endian::big(version);
 
     const auto& partVertices = req.get_parts();
-    auto spaceId = req.get_space_id();
+    spaceId_ = req.get_space_id();
     callingNum_ = partVertices.size();
+    auto iRet = schemaMan_->getTagIndexes(spaceId_);
```

Comment:
I have added an index manager maybe you can use it to replace `schemaMan_`. I will release it later.

Reply:
That's cool !!